pkg/cmd/create/cluster/scenario/create/hub/openstack/install_config.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright Contributors to the Open Cluster Management project
+
+apiVersion: v1
+metadata:
+  name: "{{ .managedCluster.name }}"
+baseDomain: {{ .managedCluster.openstack.baseDnsDomain }}
+controlPlane:
+  hyperthreading: Enabled
+  name: master
+  replicas: 3
```

Comment:
Can you support parametrization for replicas like in the other install config.
`  replicas: {{ if .managedCluster.master }}{{ .managedCluster.master.replicas }}{{ else }}3{{end}}`
I just add that parametrization.